apps/web/src/services/cost-explorer/store/cost-query-set-store.ts

```diff
@@ -30,9 +31,19 @@ export const useCostQuerySetStore = defineStore('cost-query-set', {
         },
     },
     actions: {
+        async listDataSources(): Promise<void> {
+            try {
+                const { results } = await SpaceConnector.clientV2.costAnalysis.dataSource.list();
+                this.dataSourceList = results;
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                this.dataSourceList = [];
+            }
+        },
```

Comment:
suggestion: I think it would be better to use shared store like 'data source reference store'.
I'll add it at other PR!

Reply:
Oh.. how about discussion by huddle?